flowapi/tests/unit/conftest.py

```diff
@@ -16,13 +16,24 @@
 from asyncio import Future
 
 
-def async_return(result):
+@pytest.fixture
+def async_return():
```

Comment:
Feels a bit unnecessary to have this as a fixture to me? Also I think the one time I see it used would also be achievable with https://asynctest.readthedocs.io/en/latest/asynctest.mock.html#asynctest.CoroutineMock?